package.json

```diff
@@ -25,7 +25,7 @@
     "lint": "eslint --ext=js --ext=jsx ./src",
     "lint:fix": "npm run lint -- --fix",
     "test": "jest",
-    "flow": "flow",
+    "flow": "flow check",
```

Comment:
Good catch. I have always my server running, but it should be definitely like this. 🙏